src/main/java/net/dries007/tfc/ConfigTFC.java

```diff
@@ -46,614 +31,1423 @@ public static void onConfigChangedEvent(ConfigChangedEvent.OnConfigChangedEvent
+
+        @Config.Comment("Camel")
+        @Config.LangKey("config." + MOD_ID + ".animals.camel")
+        public static final CamelCFG CAMEL = new CamelCFG();
+
+        @Config.Comment("Llama")
+        @Config.LangKey("config." + MOD_ID + ".animals.llama")
+        public static final LlamaCFG LLAMA = new LlamaCFG();
+
+        @Config.Comment("Horse")
+        @Config.LangKey("config." + MOD_ID + ".animals.horse")
+        public static final HorseCFG HORSE = new HorseCFG();
+
+        @Config.Comment("Donkey")
+        @Config.LangKey("config." + MOD_ID + ".animals.donkey")
+        public static final DonkeyCFG DONKEY = new DonkeyCFG();
+
+        @Config.Comment("Mule")
+        @Config.LangKey("config." + MOD_ID + ".animals.mule")
+        public static final MuleCFG MULE = new MuleCFG();
+
+        @Config.Comment("Ocelot")
+        @Config.LangKey("config." + MOD_ID + ".animals.ocelot")
+        public static final OcelotCFG OCELOT = new OcelotCFG();
+
+        @Config.Comment("Wolf")
+        @Config.LangKey("config." + MOD_ID + ".animals.wolf")
+        public static final WolfCFG WOLF = new WolfCFG();
+
+        @Config.Comment("Bear")
+        @Config.LangKey("config." + MOD_ID + ".animals.bear")
+        public static final BearCFG BEAR = new BearCFG();
+
+        @Config.Comment("Polar Bear")
+        @Config.LangKey("config." + MOD_ID + ".animals.polar_bear")
+        public static final PolarBearCFG POLAR_BEAR = new PolarBearCFG();
+
+        @Config.Comment("Lion")
+        @Config.LangKey("config." + MOD_ID + ".animals.lion")
+        public static final LionCFG LION = new LionCFG();
+
+        @Config.Comment("Panther")
+        @Config.LangKey("config." + MOD_ID + ".animals.panther")
+        public static final PantherCFG PANTHER = new PantherCFG();
+
+        @Config.Comment("Saber Tooth")
+        @Config.LangKey("config." + MOD_ID + ".animals.saber_tooth")
+        public static final SaberToothCFG SABER_TOOTH = new SaberToothCFG();
+
+        @Config.Comment("Hyena")
+        @Config.LangKey("config." + MOD_ID + ".animals.hyena")
+        public static final HyenaCFG HYENA = new HyenaCFG();
+
+        @Config.Comment("Deer")
+        @Config.LangKey("config." + MOD_ID + ".animals.deer")
+        public static final DeerCFG DEER = new DeerCFG();
+
+        @Config.Comment("Parrot")
+        @Config.LangKey("config." + MOD_ID + ".animals.parrot")
+        public static final ParrotCFG PARROT = new ParrotCFG();
+
+        @Config.Comment("Pheasant")
+        @Config.LangKey("config." + MOD_ID + ".animals.pheasant")
+        public static final PheasantCFG PHEASANT = new PheasantCFG();
+
+        @Config.Comment("Rabbit")
+        @Config.LangKey("config." + MOD_ID + ".animals.rabbit")
+        public static final RabbitCFG RABBIT = new RabbitCFG();
+
+        public static final class AlpacaCFG
+        {
+            @Config.Comment("How many days it is needed for this animal to be considered a fully grown adult?")
```

Comment:
I'm only going to do this one animal bit. If you want to change them, copy paste to the rest, too many to mark.

```suggestion
            @Config.Comment("How many days until this animal is a full grown adult?")
```